scripts/exps/kani-repos-utils/filter_unsupported_constructs.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+if [[ $# -ne 1 ]]; then
+  echo "$0: Error: Missing input file."
```

Comment:
Can you make this error a bit clearer? What is the input file?

Reply:
Added clarification to the error.